README.md

```diff
@@ -11,21 +11,25 @@ make
 sudo make install
 ```
 
+## Create a Changelog
+```bash
+changelog-tool new
+```
+
 ## Verify a Changelog
 ```bash
-changelog-tool verify CHANGELOG.md 
+changelog-tool verify
 ```
 ```
-verifying CHANGELOG.md...
-CHANGELOG.md is a valid changelog
+CHANGELOG.md is a valid changelog.
 ```
 
 ## Prepare a Changelog for a Release
 CHANGELOG.md:
```

Comment:
Unrelated to this PR but, the CHANGELOG here is weird. It buries the "prepare for release" command. I think we should remove/move it.

Thoughts?

Reply:
Agreed. Is the updated README more clear?